libs/RFIDKit/include/MagicCard.h

```diff
@@ -30,6 +30,8 @@ struct MagicCard {
 		return both;
 	}
 
+	[[nodiscard]] constexpr auto getLanguage() const -> uint8_t { return _tag.data.at(language_byte_index); }
```

Comment:
Instead of returning a byte (0x01, 0x02, ...) I think it is better to wrap it into an Enum to understand what is happening on use

``` cpp
enum class TagLanguage{
	None = 00,
	fr_FR = 01,
	en_US = 02
}
```

---

```cpp
	if (card.getLanguage() == TagLanguage::fr_FR) {
		_videokit.displayImage("/fs/home/img/system/robot-misc-choose_activity-fr_FR.jpg");
	} else if (card.getLanguage() == TagLanguage::en_US){
		_videokit.displayImage("/fs/home/img/system/robot-misc-choose_activity-en_US.jpg");
	}
```

Reply:
Going further, we can have a python dictionary like or struct as `Color.h` in ActivityKit to get language tag from the byte in order to directly set the right tag in the path

`_videokit.displayImage({image_system_folder} + {id_converted_name} + '-' + {tag_language} + ".jpg");`

We can abord this when we will simplify use of videokit by dropping the folder and extension, {image_system_folder} and ".jpg" in the above case.

Reply:
I agree with @YannLocatelli 

Just a small note: I would call the enum just `Language` and keep in `struct MagicCard` to access/use it via `MagicCard::Language`